core/logger/manager.php

```diff
@@ -60,6 +59,11 @@ public function rest_error_handler( $error_number, $error_message, $error_file,
 			'line' => $error_line,
 		] );
 
+		if ( ! Utils::is_elementor_path( wp_normalize_path( $error_file ) ) ) {
```

Comment:
Move the `wp_normalize_path` to `is_elementor_path`.

```suggestion
		if ( ! Utils::is_elementor_path( $error_file ) ) {
```

Reply:
Handled